File: funcscript/parser/syntax/FuncScriptParser.ParseSpaceSepratedList.cs
using FuncScript.Block;
using FuncScript.Funcs.Math;
using FuncScript.Model;

namespace FuncScript.Core
{
    public partial class FuncScriptParser
    {
        public static List<string> ParseSpaceSepratedList(ParseContext context)
        {
            var (prog, parseNode, i) = GetSpaceSeparatedStringListExpression(context,  0);
            return prog;
        }
    }
}

File: ts/core/src/parser/syntax/FuncScriptParser.ParseSpaceSepratedList.ts
import { ParseContext } from "../FuncScriptParser.Main";
import { GetSpaceSeparatedStringListExpression } from "./FuncScriptParser.GetSpaceSepratedStringListExpression";

export function ParseSpaceSeparatedList(context: ParseContext): string[] {
    //this function is not supported for the TypeScript parser
    return [];
}